const getActivities = async () => {
  try {

  } catch (error) {
    console.error(error)
  }
}